### PR TITLE
Implement initial App flow

### PR DESCRIPTION
**Pull request by ahie**

### CODE REVIEW

File: src/domain/Ahti/Ahti.module.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.menu {
+  background-color: var(--hds-ui-color-white);
+}
+
+.search {
+  width: 250px;
```
Comment (marharyta) [inline src/domain/Ahti/Ahti.module.scss]: this could be a variable

File: src/common/ui-components/Toggle/Toggle.module.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
+@import '../../../styles/icons';
+
+$toggle-margin: 5px;
+
+.container {
+  display: flex;
+  background-color: var(--hds-ui-color-black-20);
+  width: $small-icon-height * 2 + $toggle-margin * 4;
+  border-radius: 20px;
```
Comment (marharyta) [inline src/common/ui-components/Toggle/Toggle.module.scss]: same, could be a variable

Comment (ahie) [reply src/common/ui-components/Toggle/Toggle.module.scss]: Done.

File: src/domain/Ahti/Ahti.module.scss (hunk 1 above)
Comment (ahie) [reply src/domain/Ahti/Ahti.module.scss]: Done.

File: src/styles/partials/_variables.scss [hunk 3]
```diff
@@ -10,3 +10,5 @@ $image-width: 30px;
 $vertical-overflow-scrollable: scroll;
 
 $border-bottom-unset: unset;
+
+$search-component-width: 250px;
```
Comment (marharyta) [inline src/styles/partials/_variables.scss]: nice!

File: src/domain/overmind/index.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
+/* eslint-disable */
```
Comment (marharyta) [inline src/domain/overmind/index.ts]: I love everything about this!

Comment (ahie) [reply src/domain/overmind/index.ts]: Shouldn't need to be disabled. Removed.

**Pull request merged**